.bazelrc

```diff
@@ -40,9 +40,8 @@ build:release --workspace_status_command="node ./tools/bazel-stamp-vars.js"
 # Temporary Settings for Ivy   #
 ################################
 
-# Use the legacy AOT compiler strategy. We don't want to compile with Ivy nor with "ngtsc" which
-# does not generate factory files which are needed for AOT.
-build --define=compile=legacy
+# Use the Ivy AOT compiler strategy. We want to compile with Ivy and with "ngtsc".
+build --define=compile=aot --keep_going=true
```

Comment:
What are the required steps if someone wants to test something in ViewEngine?

Reply:
To build for ViewEngine, you can use `--define=compile=legacy` here instead. I think that @devversion has some ideas on how to make this more configurable and hopefully add NPM scripts to the `package.json` for building with Ivy (default) or ViewEngine.

I think that the deploy scripts will need to be changed to always build with ViewEngine since, in Angular version 9, that's the only way to publish libraries.